extensions/interactions/RatioExpressionInput/directives/ratio-expression-input-validation.service.ts

```diff
@@ -42,7 +43,7 @@ export class RatioExpressionInputValidationService {
 
   getCustomizationArgsWarnings(
       customizationArgs: RatioExpressionInputCustomizationArgs): Warning[] {
-    var isNonNegativeInt = function(number) {
+    var isNonNegativeInt = function(number: number) {
       return angular.isNumber(number) && number % 1 === 0 && number >= 0;
```

Comment:
`angular.isNumber` is an ajs method, we should not use it in Angular2+ code ? I looked for alternatives and found `isNumeric` method from rxjs, can you look into this or suggest some other alternative you find more suitable ?.

Reply:
done